openapi/Swarm.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 
 info:
-  version: 4.0.0
+  version: 4.1.0
```

Comment:
The version should not be changed here as it is already indicated in the PR description.